package.json

```diff
@@ -127,7 +125,7 @@
     "stylelint-prettier": "^2.0.0"
   },
   "devDependencies": {
-    "lerna": "^6.4.1",
+    "lerna": "^7.0.2",
```

Comment:
Updating the PR name to include "Update to Lerna 7" so it's easier to find when it was updated.

Reply:
@fcollonval curious why Lerna had to be updated here?

Is it because the GitHub actions runners bumped to Lerna 7? https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2204-Readme.md#project-management

Reply:
I bumped lerna because it was pinning an old version of semver:

![image](https://github.com/jupyterlab/jupyterlab/assets/8435071/1cd8865e-8c44-4a01-97e5-8211ad675634)


that allowed to bring only 7.5.x version of semver (to reduce security risk):

![image](https://github.com/jupyterlab/jupyterlab/assets/8435071/06f1b4de-8a62-4a6f-9dc4-505fde2072b1)